journal-of-international-economic-law.csl

```diff
@@ -0,0 +1,405 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" default-locale="en-US">
```

Comment:
you'll want to use `en-GB` as the default locale. Reason will become clear below.